Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionHTMLReportSendEmailOperations.cs

```diff
@@ -265,10 +266,29 @@ public void Execute(IReportInfo RI)
                         }
                     }
                     long emailSize = CalculateAttachmentsSize(RunSetActionHTMLReportSendEmail.Email);
-
+             
                     if (ReportItem != null)
                     {
-                        if (((EmailHtmlReportAttachment)ReportItem).IsLinkEnabled || emailSize > 10000000)
+                        
+                        if (((EmailHtmlReportAttachment)ReportItem).IsAccountReportLinkEnabled || emailSize > 10000000)
+                            {
+                            // TODO: add warning or something !!!!
+
+                            if (RunSetActionHTMLReportSendEmail.EmailAttachments.IndexOf(ReportItem) > -1)
+                                {
+                                if (RunSetActionHTMLReportSendEmail.Email.Attachments.Count > 0)
+                                    {
+                                    RunSetActionHTMLReportSendEmail.Email.Attachments.RemoveAt(RunSetActionHTMLReportSendEmail.EmailAttachments.IndexOf(ReportItem));
+                                    }
+                                }
+                            string accountReportURL = GingerRemoteExecutionUtils.GetReportHTMLServiceUrl() + "?ExecutionId=" + WorkSpace.Instance.RunsetExecutor.RunSetConfig.ExecutionID;
```

Comment:
Create function in GingerRemoteExecutionUtils to get the complete url by given Execution id

Reply:
done